crates/iota-data-ingestion-core/src/util.rs

```diff
@@ -13,15 +13,30 @@ use url::Url;
 pub fn create_remote_store_client(
     url: String,
     remote_store_options: Vec<(String, String)>,
-    timeout_secs: u64,
+    request_timeout_secs: u64,
 ) -> Result<Box<dyn ObjectStore>> {
     let retry_config = RetryConfig {
         max_retries: 0,
-        retry_timeout: Duration::from_secs(timeout_secs + 1),
+        retry_timeout: Duration::from_secs(request_timeout_secs + 1),
         ..Default::default()
     };
+
+    create_remote_store_client_with_ops(
+        url,
+        remote_store_options,
+        request_timeout_secs,
+        retry_config,
+    )
+}
+
+pub fn create_remote_store_client_with_ops(
```

Comment:
Can we add a short comment here as it's a public fun? Please also add to the other public types a brief comment just to faster understand what it's doing.

Reply:
[fixed](https://github.com/iotaledger/iota/pull/5026/commits/777681d6bcaee1d6eb20e9969aac7056bbc47eef)